app/assets/stylesheets/govuk_publishing_components/components/helpers/_brand-colours.scss

```diff
@@ -96,3 +96,27 @@
     border-color: #045f71;
   }
 }
+
+// This change should be removed after relevant govuk-frontend release
+
+.brand--department-for-business-and-trade {
+  .brand__color {
+    color: #005ea5;
```

Comment:
A small and probably not blocking comment - this colour doesn't pass WCAG AAA. It passes AA, which is good, but it would be better if it could also pass AAA. If it was adjusted to `#005A9E` it would pass AAA, but I appreciate that this may involve some back and forth.

![Screenshot 2023-04-24 at 11 25 07](https://user-images.githubusercontent.com/861310/233971693-8d1459f8-144e-443b-bb9c-d1393c66707a.png)

See https://webaim.org/resources/contrastchecker/

Reply:
@andysellick I have pushed the recent change as discussed - thank you!